internal/storage/storage.go

```diff
@@ -0,0 +1,44 @@
+package storage
+
+import (
+	"errors"
+	"urlshort/internal/shorten"
+)
+
+// NOTE: it's a temporary solution.
```

Comment:
```suggestion
// TODO: it's a temporary solution.
```
I like this comment. But let's fix this issue and exclude using global variables.